debugger/src/main/kotlin/org/rust/debugger/settings/RsDebuggerSettingsConfigurable.kt

```diff
@@ -6,33 +6,48 @@
 package org.rust.debugger.settings
 
 import com.intellij.openapi.options.SearchableConfigurable
-import com.intellij.ui.components.JBCheckBox
+import com.intellij.openapi.ui.ComboBox
+import com.intellij.ui.components.Label
 import com.intellij.ui.layout.panel
-import org.rust.openapiext.CheckboxDelegate
+import com.jetbrains.cidr.cpp.toolchains.CPPToolchains
+import org.rust.debugger.DataFormatters
 import javax.swing.JComponent
 
 class RsDebuggerSettingsConfigurable(
     private val settings: RsDebuggerSettings
 ) : SearchableConfigurable {
 
-    private val isRendersEnabledCheckbox: JBCheckBox = JBCheckBox("Enable Rust library renders")
-    private var isRendersEnabled: Boolean by CheckboxDelegate(isRendersEnabledCheckbox)
+    private val isLLDB = CPPToolchains.getInstance().defaultToolchain?.debuggerKind?.isLLDB() == true
```

Comment:
Using `CPPToolchainsConfigurable.TOPIC` is a rather tricky business, because of the lifecycle of the configurables. I'd suggest not to go that way. 
If you absolutely have to separate GDB/LLDB settings, it will be much easier to show both settings groups (for GDB and for LLDB)

Reply:
@amakeev Oh, as you are much more aware of CLion related things, let's do it in this way!
> If you absolutely have to separate GDB/LLDB settings

Maybe there is a better solution, but the current one allows us to release own pretty printers partially. Also, we still use formatters from rustup distribution as defaut variant for now.
We'll merge these settings and make bundled printers default when we implement all basic features available with formatters from rustup distribution

Reply:
Just in case: you're comment about `A user can change default toolchain` is totally valid. I meant that making the correct listener between two configurables is not straightforward.